scala-package/pom.xml

```diff
@@ -398,7 +404,7 @@
     <dependency>
       <groupId>org.scalatest</groupId>
       <artifactId>scalatest_2.11</artifactId>
-      <version>3.0.4</version>
+      <version>3.0.2</version>
```

Comment:
Any reason we downgrade the version?

Reply:
It is downgraded to use the scala version of 2.11.8